crates/swc_ecma_lexer/Cargo.toml

```diff
@@ -19,7 +19,7 @@ bench = false
 [features]
 # Used for debugging
 debug         = ["tracing-spans"]
-default       = ["typescript", "stacker"]
+default       = ["typescript", "swc_ecma_utils/stacker"]
```

Comment:
Also this should never be default

Reply:
why wouldn't this work? Shouldn't it handle large expressions correctly?